docs/concepts/workloads/controllers/cron-jobs.md

```diff
@@ -129,7 +129,7 @@ of the set of pods. A cron job does not examine pods at all.
 
 As with all other Kubernetes configs, a cron job needs `apiVersion`, `kind`, and `metadata` fields. For general
 information about working with config files, see [deploying applications](/docs/user-guide/deploying-applications),
-[configuring containers](/docs/tasks/configure-pod-container/assign-memory-resource/), and
+[configuring containers](/docs/user-guide/configuring-containers), and
```

Comment:
if the link is pointing to something that doesn't exist, I'd suggest we remove the link.

Reply:
OK,Thanks!